ecs-cli/modules/cli/configure/configure_test.go

```diff
@@ -333,3 +331,55 @@ func TestDefaultProfileDoesNotExist(t *testing.T) {
 	assert.Error(t, err, "Expected error configuring profile")
 
 }
+
+func TestMigrate(t *testing.T) {
+	configContents := `[ecs]
+cluster = defaultCluster
+aws_profile =
+region = us-west-1
+aws_access_key_id = AKID
+aws_secret_access_key = SKID
+compose-project-name-prefix =
+compose-service-name-prefix = ecs-
+cfn-stack-name-prefix =
```

Comment:
Can you add another text that `compose-service-name-prefix = ""` and `compose-service-name-prefix = amazon-ecs-cli-setup-`?